doc/source/simulating_with_spiceev.rst

```diff
@@ -191,3 +142,333 @@ CSV file options
+    |   HORIZON         |      24       | number of hours to look ahead                           |
+    +-------------------+---------------+---------------------------------------------------------+
+    |   PRICE_THRESHOLD |    0.001      | A price below this is considered cheap. Unit: € / 1 kWh |
+    +-------------------+---------------+---------------------------------------------------------+
+    |   DISCHARGE_LIMIT |      0        | V2G: maximum depth of discharge [0-1]                   |
+    +-------------------+---------------+---------------------------------------------------------+
+    |  V2G_POWER_FACTOR |      1        | Fraction of max battery power used for discharge        |
+    |                   |               | process [0-1]                                           |
+    +-------------------+---------------+---------------------------------------------------------+
+    |   LOAD_STRAT      |   "balanced   | Sub-strategies for behaviour within charging windows    |
+    |                   |               |                                                         |
+    |                   |               | (see description above for options and explanations)    |
+    +-------------------+---------------+---------------------------------------------------------+
+
+**Distributed**
+
+    +----------------------+---------------+---------------------------------------------------------------------+
+    |**Strategy option**   | **default**   |              **explanation**                                        |
+    +----------------------+---------------+---------------------------------------------------------------------+
+    |   ALLOW_NEGATIVE_SOC |   False       | simulation does not abort if SoC becomes negative                   |
+    +----------------------+---------------+---------------------------------------------------------------------+
+    |   C-HORIZON          |      3        | loading time in min reserved for vehicle if number of cs is limited |
+    +----------------------+---------------+---------------------------------------------------------------------+
+    |   DISCHARGE_LIMIT    |      0        | V2G: maximum depth of discharge [0-1]                               |
+    +----------------------+---------------+---------------------------------------------------------------------+
+    |  V2G_POWER_FACTOR    |      1        | Fraction of max battery power used for discharge                    |
+    |                      |               | process [0-1]                                                       |
+    +----------------------+---------------+---------------------------------------------------------------------+
+    |   PRICE_THRESHOLD    |    0.001      | A price below this is considered cheap. Unit: € / 1 kWh             |
+    +----------------------+---------------+---------------------------------------------------------------------+
+
+.. _file_formats:
```

Comment:
Ok for now since it is just a restructuring. I think later we can delete this part or shorten it quite a bit.

Reply:
to be done later